test/parallel/test-child-process-execfile.js

```diff
@@ -99,3 +99,15 @@ const execOpts = { encoding: 'utf8', shell: true };
   });
   execFile(process.execPath, [fixture, 0], { signal }, callback);
 }
+
+// Verify the execFile() stdout is the same as execFileSync().
+{
+  const execArgs = ['echo', ['foo', 'bar'], { shell: true, encoding: 'utf8' }];
+  const execFileSyncResult = execFileSync(...execArgs);
+  assert.strictEqual(execFileSyncResult.trim(), execArgs[1].join(' '));
```

Comment:
Can you also check that the output is "foo bar"? Ideally, test with and without `{ shell: true }`.